java-quickstart/build.gradle

```diff
@@ -4,7 +4,8 @@ dependencies {
 
     compile group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
     compile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.12'
-    compile group: 'org.apache.logging.log4j', name: 'log4j-slf4j-impl', version: '2.13.3'
+    compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
```

Comment:
I see this is a revert to the line that was there previously. That works, tested it. But I'm not sure if maybe new Apache logger might have different logging properties setup, and this line didn't need to be reverted and instead Apache logger might need a different setup. I'm not sure.

Reply:
No actually previously it was     _compile group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.30'_  which was causing vulnerability issue due to transitive property of log4j. I now replaced it with alternative library  slf4j-simple.